gef-ui/server/controllers/facility-guarantee/facility-guarantee.js

```diff
@@ -23,15 +25,19 @@ function validateFacilityGuarantee({
     errMsg: 'Select how often your bank will pay the fee to UKEF',
   };
 
-  if (!['in advance', 'in arrears', 'at maturity'].includes(feeType)) {
+  if (![
+    CONSTANTS.FACILITY_PAYMENT_TYPE.IN_ADVANCE,
+    CONSTANTS.FACILITY_PAYMENT_TYPE.IN_ARREARS,
+    CONSTANTS.FACILITY_PAYMENT_TYPE.AT_MATURITY,
+  ].includes(feeType)) {
```

Comment:
for readability, may it be easier to do the same as in line 35? where array defined as variable and checking if variable includes?

Reply:
Great idea, updated 👍